Xamarin.PropertyEditing.Mac/Controls/PointEditorControl.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System;
+using System.Collections;
+using System.Diagnostics;
+using System.Drawing;
+using AppKit;
+using CoreGraphics;
+using Xamarin.PropertyEditing.ViewModels;
+
+namespace Xamarin.PropertyEditing.Mac
+{
+	internal class PointEditorControl : PropertyEditorControl
+	{
+		internal NSTextField XEditor { get; set; }
```

Comment:
@ermau they don't have to be, but I was trying to be consistent with how the previous controls set up their NS* controls. If we change it here, then I think we should do a pass and remove them in all the controls.